### ISSUE TITLE
Create Endpoint for saving popcorn edit revisions on Mediawiki

### ISSUE THREAD

**Issue by Nolski** [mozilla/popcorn-editor#21]

**Comment by traceypooh** [comment]
hey so is saving an edited project possible at all anwhere yet?  (hi from archive.org :) )

**Comment by traceypooh** [comment]
(ah, may have answered my own question 8-) seems like for now save does console.log post of the project JSON, etc...)

**Comment by Nolski** [comment]
Yup! The "save API" is something that you'll have to write your own front end code against to send that data to be saved on your server. This issue is specifically for MediaWiki integration. (By the way I didn't get a chance to say hi but I think I saw you at MozFest last month :) )
